### DISCUSSION TITLE
Getting user solutions via API

### DISCUSSION THREAD

**Discussion by AlexRyabikov** [codewars/codewars.com#3106] [Ideas]
Several days ago I've tried to use codewars API and all was ok except getting my solutions of katas. There is no method for it yet. Only possible way:  use little help of external libraries. All other things: getting urls, names of katas, descriptions and ranks - works fine. 

How do you think, will it be a useful function of codewars api?

**Comment by hobovsky** [comment]
It's already been requested in [this ticket](https://github.com/codewars/codewars.com/issues/2305) and other, similar tickets.

**Reply by hobovsky** [reply]
I'm closing this discussion as a duplicate of #2305 .
